packages/vega-typings/types/spec/stream.d.ts

```diff
@@ -21,6 +22,96 @@ export type EventType =
   | 'touchmove'
   | 'touchstart'
   | 'wheel';
+
+export type HTMLBodyEventType =
+  | EventType
+  | 'abort'
```

Comment:
Can we use `keyof GlobalEventHandlersEventMap` instead of the list here?

Reply:
Referencing to `GlobalEventHandlersEventMap` is the reason why the schema generator fails. It would result in
```bash
UnknownNodeError: Unknown node "GlobalEventHandlers" (ts.SyntaxKind = 165)
```

Reply:
What is syntax kind 165?

Reply:
From the linked issue vega/vega-lite#4620:

> Currently it does not have support for SyntaxKind = 165, which is FunctionType.